### PR TITLE
Publish toggle

### PR DESCRIPTION
**Pull request by Joozty**

### CODE REVIEW

File: components/published-toggle/index.css [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+}
+
+.toggle-switch::before {
+  background: var(--icon-published) no-repeat 5px 50%;
+
+  /** TODO: Text doesn't show properly
+    mask-image: var(--icon-unpublished);
+    mask-repeat: no-repeat;
+    mask-position: 5px 50%;
+  */
```
Comment (viktor-yakubiv) [inline components/published-toggle/index.css]: Try multiple backgrounds

Comment (Joozty) [reply components/published-toggle/index.css]: Unfortunately, it didn't work. :(

**Comment by Joozty** [comment]
@viktor-yakubiv It looks ugly but the functionality is in there. We can play with styles later.

**Comment by viktor-yakubiv** [comment]
@Joozty do you want me to merge or you would like (me) to work on it more?

**Comment by Joozty** [comment]
I changed the colours and animation. If you want me to work on it more please tell me what's should be improved.

**Pull request merged**